gpkit/solution_array.py

```diff
@@ -592,6 +593,25 @@ def savetxt(self, filename="solution.txt", *, printmodel=True, **kwargs):
                 f.write(self.modelstr + "\n")
             f.write(self.table(**kwargs))
 
+    def savejson(self, filename="solution.json"):
```

Comment:
Looks great! There should be a test that just verifies the output of this on e.g. the Peformance Modeling example...will add it when I can or you're welcome to take a stab!

Reply:
Test is up under the performance modeling example in t_examples.py. Let me know if this is the kind of test you would be looking for - my stab may have been more like a gouge.

Reply:
looks great to me